contracts/core/strategy/StrategyExecutor.sol

```diff
@@ -101,4 +102,36 @@ contract StrategyExecutor is StrategyModel, AdminAuth, CoreHelper, CheckWalletTy
             )
         );
     }
+
+
+    //////////////////// ONE TIME FUNDS RESCUE /////////////////////////
+
+    /// @dev User set his own proxy.owner() to proxy address, custom logic to override the owner to new account
+    /// @dev Only callable for this specific user and as long as .owner() == proxy_address
+    function recoverOwner() external {
+        // hardcoded address used for specific user
+        address changeOwnerActionAddr = 0x81cA52CfE66421d0ceF82d5F33230e43b5F23D2B;
+        address userWallet = 0xddc65fAC7201922395045FFDFfe28d3CF6012E22;
+        address userNewEOA = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE; // TODO: TBD
```

Comment:
put owner address here so even if we accidentally deploy it without changes we do not brick even harder